SimPPS/DirectSimProducer/python/ppsDirectProtonSimulation_cff.py

```diff
@@ -0,0 +1,37 @@
+import FWCore.ParameterSet.Config as cms
+from CalibPPS.ESProducers.ctppsBeamParametersFromLHCInfoESSource_cfi import ctppsBeamParametersFromLHCInfoESSource as _esLHCinfo
+from SimPPS.DirectSimProducer.ppsDirectProtonSimulation_cfi import ppsDirectProtonSimulation as _dirProtonSim
+from IOMC.EventVertexGenerators.beamDivergenceVtxGenerator_cfi import beamDivergenceVtxGenerator as _vtxGen
+
+# vertex smearing
+beamDivergenceVtxGenerator = _vtxGen.clone()
+
+# beam parameters as determined by PPS
+ctppsBeamParametersFromLHCInfoESSource = _esLHCinfo.clone(
+    lhcInfoLabel = cms.string(""),
+    # beam divergence (rad)
+    beamDivX45 = cms.double(30.e-6),
+    beamDivX56 = cms.double(30.e-6),
+    beamDivY45 = cms.double(30.e-6),
+    beamDivY56 = cms.double(30.e-6),
+    # vertex offset (cm)
+    vtxOffsetX45 = cms.double(0.),
+    vtxOffsetX56 = cms.double(0.),
+    vtxOffsetY45 = cms.double(0.),
+    vtxOffsetY56 = cms.double(0.),
+    vtxOffsetZ45 = cms.double(0.),
+    vtxOffsetZ56 = cms.double(0.),
+    # vertex sigma (cm)
+    vtxStddevX = cms.double(1.e-3),
+    vtxStddevY = cms.double(1.e-3),
+    vtxStddevZ = cms.double(5.)
+)
+
+# direct simulation
+ppsDirectProtonSimulation = _dirProtonSim.clone(
+    hepMCTag = cms.InputTag('beamDivergenceVtxGenerator'),
+    pitchStrips = cms.double(66.e-3 * 12 / 19),  # effective value to reproduce real RP resolution
+    pitchPixelsHor = cms.double(50.e-3),
+    pitchPixelsVer = cms.double(80.e-3),
+    produceScoringPlaneHits = cms.bool(False),
```

Comment:
```suggestion
    hepMCTag = 'beamDivergenceVtxGenerator',
    pitchStrips = 66.e-3 * 12 / 19,  # effective value to reproduce real RP resolution
    pitchPixelsHor = 50.e-3,
    pitchPixelsVer = 80.e-3,
    produceScoringPlaneHits = False,
```

Reply:
Can we safely replace a `cms.InputTag` by a standard python string?

Reply:
Worth to try and verify the output (perhaps quotes not needed here...)
I am also not sure about the math operation performed at L33, also worth to check.
In case of troubles, one can stick on the un-safe syntax for those particular parameters

Reply:
> Can we safely replace a `cms.InputTag` by a standard python string?

Yes, in case only the module label part needs to be set. If also the product instance and/or process name need to be set, one can use colon-separated string `pset.tag = "module:instance:process"` or a tuple `pset.tag = ("module", "instance", "process")`.